internal/docstore/dynamodocstore/query.go

```diff
@@ -385,3 +385,26 @@ func (it *documentIterator) Stop() {
 	it.items = nil
 	it.last = nil
 }
+
+func (c *collection) QueryPlan(q *driver.Query) (*driver.QueryPlan, error) {
+	if c.description == nil {
```

Comment:
Does c need locking?

Reply:
I just moved it to the constructor instead. Much simpler.